backend/src/main/java/de/neuefische/smartcount/SmartCountController.java

```diff
@@ -20,8 +21,8 @@ public Expense createExpense(@RequestBody Expense expense) {
     }
 
     @GetMapping
-    public Collection<Expense> getAllExpenses() {
-        return smartCountService.getExpenses();
+    public ExpenseDTO getExpensesDTO() {
+        return smartCountService.getExpensesDTO();
```

Comment:
```suggestion
        return return new ExpenseDTO(smartCountService.getExpenses(), smartCountService.getSum());;
```
See comment in `SmartCountService` for more details.